src/frontend/staff-portal/src/app/app.module.ts

```diff
@@ -56,9 +56,9 @@ function initializeKeycloak(keycloak: KeycloakService): () => Promise<void> {
   return async() => {
     await keycloak.init({
       config: {
-        url: 'https://dev.oidc.gov.bc.ca/auth',
```

Comment:
this code from me initially works to point to the dev version of the keycloak server, however, the code will need to be changed since we now have multiple keycloak servers, one for dev, one for test, the server name cannot be hard coded.

Reply:
ok, I will change this part

Reply:
you have set the clientId to environment.keycloakRealm lin 55